src/FrameRipper.py

```diff
@@ -0,0 +1,79 @@
+import cv2
+import numpy as np
+import os 
+
+#Frame Ripper Script WIP
+#Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+#
+#Steve
+
+#Function to scan the video files, processes and adds them to Array video_list
+def getVideoFiles():
+    
+    entries = os.listdir('Videos')
+    video_list = []
+    
+    for entry in entries:
+        video_list.append("Videos/" + entry)
+    return video_list
+
+#Function that takes each element from the video_list and returns the total frame count of each video file
+def getTotalFrames():
+    
+    videos = getVideoFiles()
+    total_frames = []
+    
+    for index, file_name in enumerate(videos):
+        cap = cv2.VideoCapture(file_name)
+        frameCount = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+        total_frames.append(frameCount)
+    return total_frames
+
+#Master Fucntion: Rips frames and creates seperate directories for each individual video
+def ripFrames():
```

Comment:
I have learned somewhere that you should keep functions/methods to at most 25 lines. That way it doesn't extend on so far, no scrolling, just the method contained in a small block for easy reading.